python/dgllife/model/model_zoo/potentialnet.py

```diff
@@ -0,0 +1,223 @@
+    n_rows_fc: list of int
+        The widths of the fully connected neural networks at each layer in stage 3.
+    dropouts: list of 3 floats
+        The amount of dropout applied at the end of each stage.
+    """
+    def __init__(self,
+                 n_etypes,
+                 f_in,
+                 f_bond,
+                 f_spatial,
+                 f_gather,
+                 n_rows_fc,
+                 n_bond_conv_steps,
+                 n_spatial_conv_steps,
+                 dropouts
+                 ):
+        super(PotentialNet, self).__init__()
+        self.stage_1_model = Customized_GatedGraphConv(in_feats=f_in,
+                                        out_feats=f_bond,
+                                        f_gather=f_gather,
+                                        n_etypes=5,
+                                        n_steps=n_bond_conv_steps,
+                                        dropout=dropouts[0]
+                                        )
+        self.stage_2_model = Customized_GatedGraphConv(in_feats=f_gather,
+                                        out_feats=f_spatial,
+                                        f_gather=f_gather,
+                                        n_etypes=n_etypes, # num_distance_bins + 5 covalent types
+                                        n_steps=n_spatial_conv_steps,
+                                        dropout=dropouts[1]
+                                        )
+        self.stage_3_model = StagedFCNN(f_in=f_gather,
+                                        n_row=n_rows_fc,
+                                        dropout=dropouts[2]
+        )
+
+    def forward(self, bigraph, knn_graph):
+        batch_num_nodes = bigraph.batch_num_nodes()
+        h = self.stage_1_model(graph=bigraph, feat=bigraph.ndata['h'])
+        h = self.stage_2_model(graph=knn_graph, feat=h)
+        x = self.stage_3_model(batch_num_nodes=batch_num_nodes, features=h) 
+        return x
+
+class StagedFCNN(nn.Module):
+    """
+    The implementation of PotentialNet stage 3.
+    A graph gather is performed solely on the ligand atoms followed a multi-layer fully connected neural networks.
+    """
+    def __init__(self,
+                 f_in,
+                 n_row,
+                 dropout):
+        super(StagedFCNN, self).__init__()
+        self.dropout = nn.Dropout(p=dropout)
+        self.layers = nn.ModuleList()
+        self.layers.append(nn.Linear(f_in, n_row[0]))
+        for i in range(1, len(n_row)):
+            self.layers.append(nn.Linear(n_row[i-1], n_row[i]))
+        self.out_layer = nn.Linear(n_row[-1], 1)
+
+    def forward(self, batch_num_nodes, features):
+        x = sum_ligand_features(features, batch_num_nodes)
+        for i, layer in enumerate(self.layers):
+            if i !=0:
+                x = self.dropout(x)
+            x = layer(x)
+            x = F.relu(x)
+        x = self.out_layer(x)
+        return x
+
+class Customized_GatedGraphConv(nn.Module):
```

Comment:
We follow the camel case naming convention for classes. That says, `Customized_GatedGraphConv` should be `CustomizedGatedGraphConv` instead.